### ISSUE TITLE
Add support for arm64 machines

### ISSUE THREAD

**Issue by colichri** [OpenSlides/OpenSlides#6718]
### Is your feature request related to a problem? Please describe.
Unfortunately, I haven't seen any requirements or specifications in the documentation a server should meet when deploying Openslides on your own. So I got ready a bare arm64 server and managed to install the Openslides Manage Service but got thrown an error upon executing the executable:
`-bash: ./openslides: cannot execute binary file: Exec format error` 
As you might already guess, it is because the executable is compiled for x86_64.
Glad that the Manage Service had binary files available, I successfully managed to build the executable with the makefile.

After pulling the images with `docker compose pull` and starting the container, I faced the same problem again:
`The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested`
I then realised, that the pulled image is also comprised of 11 other compiled images, and I wouldn't get to run Openslides in an acceptable period of time.

### Describe the solution you'd like
- A list of specifications a self-hosted environment should meet to properly run Openslides to prevent other users from encountering the same problem only during the installation process
- Instructions on how to compile the binaries to make them executable on an arm64 system OR
- provide an additional arm64 image

### Describe alternatives you've considered
Re-compiling the files on the Openslides main branch to make them executable on my arm64 system, but I'm afraid of doing something I haven't got sufficient knowledge about. Friends of mine discouraged me from doing this because Openslides doesn't seem to be intended for working on arm64 machines.

**Comment by jsangmeister** [comment]
Unfortunately, we do not plan on supporting arm64 processors since the ressources requirements to run OpenSlides are typically not met by these. But it should be feasible for you to build the images manually for each repository. If you clone this repository with all subrepos and change the base image in every `Dockerfile` in every subrepo to an arm64 compatible one, you should be able to run it. Another helpful point might be the `/dev/localprod/` folder which contains scripts to run a productive setup build from your local files. I hope this helps you.

**Comment by colichri** [comment]
Thank you very much for your response. I got it up and running with the instructions you provided.
I might mention the steps I went through

- Add the parameter `--platform=linux/arm64/v8` after every `FROM` command (e.g. `FROM --platform=linux/arm64/v8 node:20.10-alpine AS build`) in each Dockerfile included in the Git subrepos.
- Build the Manage Service Tool as stated in the applicable [documentation](https://github.com/OpenSlides/openslides-manage-service/tree/454c523c31158580e375984c9de504fddf19f227)
- Copy the executable in the `/dev/localprod` directory (e.g. `sudo cp openslides ../dev/localprod`)
- Run the setup as stated in the applicable [documentation](https://github.com/OpenSlides/OpenSlides/tree/main/dev/localprod)
- Run the `docker-compose up --build` command and let the magic happen (took me 497.3 seconds)
- Enjoy!
